src/wp-includes/media.php

```diff
@@ -3882,7 +3882,11 @@ function wp_prepare_attachment_for_js( $attachment ) {
 	if ( isset( $meta['filesize'] ) ) {
 		$bytes = $meta['filesize'];
 	} elseif ( file_exists( $attached_file ) ) {
-		$bytes = filesize( $attached_file );
+		$meta = wp_generate_attachment_metadata( $attachment->ID, $attached_file );
```

Comment:
https://github.com/WordPress/wordpress-develop/pull/671/files#r515846393

Reply:
`wp_generate_attachment_metadata()` should not be used here, same as above.